microsoft-edge/hosting/webview2/webview2-api-browser.md

```diff
@@ -0,0 +1,19 @@
+---
+description: Host web content in your Win32 app with the Microsoft Edge WebView 2 control
+title: Microsoft Edge WebView 2 for Win32 apps
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 04/09/2020
+ms.topic: reference
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
+---
+
+# WebView2 API browser
+
+Welcome to the Microsoft Edge WebViewe2 API browser. Start with selecting the language you would like to work with.
```

Comment:
@pagoe-msft Please advise the proper word to use here. Thanks!

Reply:
WebViewe2 --> WebView2

WebView2 API browser --> WebView2 API Reference

paragraph:

The Microsoft Edge WebView2 control enables you to host web content in your application using Microsoft Edge (Chromium) as the rendering engine. See WebView2 for an overview and Getting Started for a quick start. 

Select the framework of WebView2 you would like to use below:

Frameworks
...